MinPlatformPkg/Test/Library/PlatformConfigCheckLibNull/DxePlatformConfigCheckLibNull.inf

```diff
@@ -0,0 +1,28 @@
+## @file
+# Platform configuration check library NULL implementation for TestPointCheckLib
+#
+# Copyright (c) Microsoft Corporation. All rights reserved
+#
+##
+
+[Defines]
+  INF_VERSION                    = 1.27
+  BASE_NAME                      = DxePlatformConfigCheckLib
+  FILE_GUID                      = ABA05EC8-A7C3-4198-B8ED-4A0D6B9F8296
+  MODULE_TYPE                    = DXE_DRIVER
```

Comment:
This NULL library instance should be a BASE type unless there's a specific reason it cannot be.

```suggestion
  MODULE_TYPE                    = BASE
```